requirements.txt

```diff
@@ -0,0 +1,3 @@
+requests >= 2.6.1
```

Comment:
Why adding such a permisive requirement? Even requests 2.20 had some serious security vulnerability, we definitely shouldn't permit anything below 2.21

Reply:
ohh i was testing action in github
action needed requirements.txt and from https://github.com/Anorov/cloudflare-scrape/blob/master/setup.py i toke install_requires=["requests >= 2.6.1"]

Reply:
min i will roll back

Reply:
ok i rolled it back now, was trying to add check action in checks tab that is between Commits "Files changed"